recipes/jungle/all/conandata.yml

```diff
@@ -0,0 +1,12 @@
+sources:
+  "cci.20220801":
+    url: "https://github.com/eBay/Jungle/archive/f41b7123489f1bc942a6b76dc54485391485cd27.tar.gz"
+    sha256: 8667a114bcef661b2a93e627a68b0584931f182dc8b96693ce6901d903584ab8
+patches:
+  "cci.20220801":
```

Comment:
Please, update patches section, adding a short description and its type, like described:  https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches

Reply:
:+1: